e2e/rbd.go

```diff
@@ -366,6 +367,56 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and bind it to an app with encrypted RBD volume with VaultTokensKMS", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				scOpts := map[string]string{
+					"encrypted":       "true",
```

Comment:
encryptionKMSType is not needed here?

Reply:
no, that is fetched from the `config.json` by looking for the `encryptionKMSID` key